bazel/external_crates.bzl

```diff
@@ -735,7 +735,7 @@ def external_crates_repository(name, cargo_lockfile, lockfile, sanitizers_enable
                 version = "^0.2.155",
             ),
             "libflate": crate.spec(
-                version = "^1.1.2",
+                version = "^2.0.0",
```

Comment:
Why I'm bumping this dependency version in this PR: 

This version 2.0.0 corresponds to what is currently used in ic-wasm 0.8.0